src/Equinox/Tracing.fs

```diff
@@ -0,0 +1,35 @@
+module Equinox.Core.Tracing
+
+open System.Diagnostics
+
+let source = new ActivitySource("Equinox")
+
+type private Activity with
+
+    member act.AddLeader(requiresLeader) =
+        if requiresLeader then act.SetTag("eqx.requires_leader", true) else act
+
+    member act.AddRetryAttempt(attempt: int) =
+        if attempt > 1 then act.SetTag("eqx.retry_count", attempt - 1) else act
+
+    member act.AddSyncAttempt(attempt: int) =
+        if attempt > 1 then act.SetTag("eqx.resync_count", attempt - 1) else act
+
+    member act.AddStale(maxAge: System.TimeSpan) =
+        if maxAge.Ticks <> 0L then act.SetTag("eqx.allow_stale", true) else act
```

Comment:
should this become a measure in ms or something?

Reply:
Yep, added both!